### PR TITLE
fix: errors thrown from loader-runner not caught/handled

### PR DESCRIPTION
**Pull request by valery-vitko**
Not quite sure it should go as is, but could be used as a base for the fix.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -254,14 +254,17 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 									compilation.errors.push(err);
```
Comment (Kovensky) [inline index.js]: The actual problem is that `compilation.rebuildModule` does not call the callback with an `err` even if there was a compilation failure -- when the `module.meta[NS]` is unset bug is hit, `module.errors` is (AFAICT) always non-empty.

**Comment by Zyphrax** [comment]
It would be so great to have this fixed. It is the only thing preventing me from using the latest version.

**Comment by bebraw** [comment]
Can you sign the CLA and rebase?

**Comment by michael-ciniawsky** [comment]
Fixes #218

**Comment by michael-ciniawsky** [comment]
@valery-vitko Please close and reopen the PR to trigger the CLA Bot again and sign the CLA, then rebase against current master 😛

File: index.js [hunk 2]
```diff
@@ -276,15 +276,18 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
...
 									return callback();
 								}
-								meta = module[NS];
-								// Error out if content is not an array and is not null
-								if(!Array.isArray(meta.content) && meta.content != null) {
-									err = new Error(module.identifier() + " doesn't export content");
-									compilation.errors.push(err);
-									return callback();
+								// TODO: Investigate "TypeError: Cannot read property '...\node_modules\extract-text-webpack-plugin' of null" in case of css errors and remove workaround check for module.meta[NS] below
```
Comment (michael-ciniawsky) [inline index.js]: Is this done or still outstanding ?

Comment (valery-vitko) [inline index.js]: Are you talking about TypeError investigation? No, but the error this workaround aims hasn't been popping out anymore for quite a while now.

Comment (michael-ciniawsky) [inline index.js]: hmm... 😛 we would still need a test for this I guess, if you don't have this regression anymore (not urgent for you), mabe better close the PR ?

Comment (valery-vitko) [inline index.js]: Yes, it don't bugs me, but it seems it's gone for good for everyone. Thus not quite sure how is it better to test this case now, though I'm not an expert in this stuff.
So, closing this PR then.

**Pull request closed without merging**